libazureinit/src/provision/ssh.rs

```diff
@@ -37,17 +39,80 @@ pub(crate) fn provision_ssh(
     Ok(())
 }
 
+pub(crate) fn get_authorized_keys_path_from_sshd() -> io::Result<Option<PathBuf>>
+{
+    let sshd_output = Command::new("sshd").arg("-G").output()?;
```

Comment:
-G doesn't work for me on Ubuntu. Use -T?

Reply:
Used -T for sshd config